packages/dapp/src/components/Navbar/Menu/ChainsMenu/ChainsMenu.tsx

```diff
@@ -44,13 +37,15 @@ export const ChainsMenu = ({ handleClose, open }: NavbarMenuProps) => {
           <MenuItemComponent
             key={`${el.label}-${index}`}
             label={el.label}
-            showButton={el.showButton}
+            showButton={el.showButton ? el.showButton : false}
             showMoreIcon={false}
-            suffixIcon={el.chainId === account.chainId && <CheckIcon />}
+            suffixIcon={
+              (el.chainId && el.chainId === account.chainId && <CheckIcon />) ||
+              undefined
+            }
```

Comment:
Let's make it more reactish way ->

```ts
el.chainId && el.chainId === account.chainId ? <CheckIcon /> : null
```

Reply:
Thanks! Applied your code but replaced the "null" with `undefined`